src/Build/Evaluation/Evaluator.cs

```diff
@@ -2725,6 +2743,25 @@ private static string StringifyList(IList<string> strings)
 
             return sb.ToString();
         }
+
+        private void SetAllProjectsProperty()
+        {
+            if (_lastModifiedProject != null)
+            {
+                P oldValue = _data.GetProperty(Constants.MSBuildAllProjectsPropertyName);
+
+                P newValue = _data.SetProperty(
+                    Constants.MSBuildAllProjectsPropertyName,
+                    oldValue == null
+                        ? _lastModifiedProject.FullPath
+                        : String.Format(CultureInfo.CurrentCulture, "{0};{1}", _lastModifiedProject.FullPath, oldValue.EvaluatedValue), isGlobalProperty: false, mayBeReserved: false);
```

Comment:
Do we want `CurrentCulture`? Shouldn't it be invariant?

Reply:
All other places in this file use `CurrentCulture`...

Reply:
I'd change it. And just use string interpolation rather than `string.Format`.